README.md

```diff
@@ -9,21 +9,29 @@ to setup API-tests in your local systems.
 
 1. Fetch the Repo: `go get -u -v github.com/Percona-Lab/pmm-api-tests`
 2. Navigate to the tests root folder:  `cd ~/go/src/github.com/Percona-Lab/pmm-api-tests`
-3. `make`
 
 # Usage
 
-Once the binaries for the tests have been generated, look at the usage of the tests using:
+Run the tests using the following command:
+
 ```
-./inventory.test -h
+go test ./... -pmm.server-url **pmm-server-url** -v
 ```
 
-It should provide a list of options that are available for execution.
+where `pmm-server-url` should be pointing to pmm-server.
 
-Run the tests using the following command:
+# Docker
+
+Build Docker image using the following command:
 
 ```
-./inventory.test -pmm.server-url **pmm-server-url** -test.v
+docker build -t IMAGENAME .
 ```
 
-where `pmm-server-url` should be pointing to pmm-server.
+Run Docker container using the following command:
+
+```
+docker run -e PMM_SERVER_URL=**pmm-server-url** IMAGENAME
```

Comment:
I don't think it will work, maybe I am wrong, but a PMM-server running on another docker container, can not be accessed by another docker container, unless, I link them together?

Reply:
Yes, you are right, but I don't think that we should write about linking container or using network=host. @AlekSi what do you think?

Reply:
We (now?) have text about `--network=host` just below.

Reply:
yes, I added that after this comment.